sparta/CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ if (NOT rc EQUAL "0")
 endif ()
 
 if (USE_LTO)
-  set (LTO_FLAGS "-flto -Ofast")
+  add_compile_options (-flto -Ofast)
```

Comment:
hmm, is this on for all variants? Release, Debug and RelWithDebInfo?

Reply:
Yeah, probably not.  I think I'll remote this for sparta.  The simulators that use it can define it on their own...

Reply:
You can specialize based on [CMAKE_BUILD_TYPE](https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html), so debug gets -O0 and -ggdb3 and RelWithDebInfo has O3.

Reply:
Yep.  Did that on an internal project.  I'm hesitant to play around with it here.  In my experiments, LTO didn't buy much at the Sparta level, but definitely at the simulation level.  Mostly because Sparta is headers...

Reply:
seems fair, I guess I was thinking whether sparta's release variant got O3 or O2, the cmake default. in either case I'm not worried : )